README.md

```diff
@@ -24,12 +24,36 @@ Or if you want to use the latest Feathr code from GitHub:
 pip install git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project
 ```
 
-## Quick Start
+## Running Feathr with only a few clicks
 
-- Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb) to try it out. There is also a companion [quick start guide](./docs/quickstart.md) containing a bit more explanation on the notebook.
-- For more details, read our [documentation](https://linkedin.github.io/feathr/).
+Feathr has native cloud integration and getting started with Feathr is very straightforward. You only need three steps:
 
-## Defining Features with Transformation
+1. Get the principal ID of your account by running `az ad signed-in-user show --query objectId -o tsv` in the link below (Select "Bash" if you are asked to choose one), and write down that value (will be something like `b65ef2e0-42b8-44a7-9b55-abbccddeefff`)
+
+
+[![Launch Cloud Shell](https://shell.azure.com/images/launchcloudshell.png "Launch Cloud Shell")](https://shell.azure.com/bash)
+
+2. Click the button below to deploy a minimal set of Feathr resources. Note that you should have "Owner" access in your subscription to perform some of the actions, and if you don't, please ask your IT admin to use [this quick start guide](./docs/how-to-guides/azure-deployment.md) to provision a service principal for you to use.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Flinkedin%2Ffeathr%2Fone_click_deployment%2Fdocs%2Fhow-to-guides%2Fazure_resource_provision.json)
+
+
+3. Run the Jupyter Notebook by clicking the button below. The only thing you need to fill in is the resource prefix you provided in the second step.
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/linkedin/feathr/main?labpath=feathr_project%2Ffeathrcli%2Fdata%2Ffeathr_user_workspace%2Fnyc_driver_demo.ipynb)
+
+## Running Feathr Examples
+
+- Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb) to try it out. There is also a companion [quick start guide](./docs/quickstart.md) containing a bit more explanation on the notebook. 
+- We recommend using **Visual Studio Code**, or **Azure Machine Learning Service** to run the above notebook, since those environments will help you login and reterive necessary credentials.
+
+## Documentation
+
+For more details, read our [documentation](https://linkedin.github.io/feathr/).
+
+
+## Feathr Capabilities
+### Defining Features with Transformation
```

Comment:
i prefer we keep the homepage read me clean and move this to a quickguide or docs/readme.